<dec f='include/python3.5m/pyerrors.h' l='391' type='PyObject * _PyUnicodeTranslateError_Create(PyObject * object, Py_ssize_t start, Py_ssize_t end, const char * reason)'/>
